libraries/standard/http/cbmc/proofs/Makefile-project-defines

```diff
@@ -0,0 +1,32 @@
+# -*- mode: makefile -*-
+# The first line sets the emacs major mode to Makefile
+
+################################################################
+# Use this file to give project-specific definitions of the command
+# line arguments to pass to CBMC tools like goto-cc to build the goto
+# binaries and cbmc to do the property and coverage checking.
+#
+# Use this file to override most default definitions of variables in
+# Makefile.common.
+################################################################
+
+# Flags to pass to goto-cc for compilation (typically those passed to gcc -c)
+COMPILE_FLAGS += \
+        -fPIC \
+        -std=gnu90 \
+
+# Flags to pass to goto-cc for linking (typically those passed to gcc)
+# LINK_FLAGS =
+
+# Preprocessor include paths -I...
+INCLUDES += \
+        -I$(SRCDIR)/libraries/standard/http/cbmc/include \
+        -I$(SRCDIR)/libraries/standard/http/include \
+        -I$(SRCDIR)/libraries/standard/http/src \
+        -I$(SRCDIR)/libraries/standard/http/third_party/http_parser \
+        -I$(SRCDIR)/demos/logging-stack \
+        -I$(SRCDIR)/demos/http/http_demo_plaintext \
+
+# Preprocessor definitions -D...
+DEFINES += \
+        -Dhttp_EXPORTS \

```

Comment:
Please consider using the following style.  I saw some evidence when running your proofs on my mac that the line continuation style may not interact well with the macros in the Makefile.common style my team has developed.

```
COMPILE_FLAGS += -fPIC 
COMPILE_FLAGS += -std=gnu90 

INCLUDES += -I$(SRCDIR)/libraries/standard/http/cbmc/include 
INCLUDES += -I$(SRCDIR)/libraries/standard/http/include 
INCLUDES += -I$(SRCDIR)/libraries/standard/http/src 
INCLUDES += -I$(SRCDIR)/libraries/standard/http/third_party/http_parser 
INCLUDES += -I$(SRCDIR)/demos/logging-stack 
INCLUDES += -I$(SRCDIR)/demos/http/http_demo_plaintext 

DEFINES += -Dhttp_EXPORTS 
```